server/src/Utils/InterfacesIntegration.ts

```diff
@@ -0,0 +1,14 @@
+import { httpRequest } from '../HTTPRequest';
+
+require('dotenv').config();
+
+const apiUrl = process.env.ENVIRONMENT === 'prod' ? process.env.INTERFACES_INTEGRATION_API_PROD : process.env.INTERFACES_INTEGRATION_API_TEST;
```

Comment:
why need both URLs? do if there is an environment to post the data then send it to the API url